### DISCUSSION TITLE
blog-2022-03-10-quanto-dinheiro-voce-pode-tirar-conta-pj

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#10] [Fan Comments]
# blog-2022-03-10-quanto-dinheiro-voce-pode-tirar-conta-pj

Você quer transferir o dinheiro da sua conta jurídica, mas o medo de transferir e a receita te prender é maior? É, eu também estive nessa situação.

https://mrpowergamerbr.com/br/blog/2022-03-10-quanto-dinheiro-voce-pode-tirar-conta-pj/

**Comment by brunorsch** [comment]
Power, se possível, continua postando esses conteúdos relacionados a "Como é ter uma empresa no Brasil". Eu tb sou um dev, e não um contador, mas queria abrir uma empresa e muitas das dúvidas que eu tenho, não sei pesquisar e não encontro a resposta pontual. Teus posts ajudam muito nesse processo, e acho que tu poderia continuar postando mais coisas que não é tão fácil assim de achar na net a resposta.

Eu tb sugeriria replicar o conteúdo (Não a integra, só "o básico") pro Medium e linkar o post original do blog, talvez lá tu acabe ajudando ainda mais gente que tá acostumada a buscar coisas pra ler por lá. Não sei se tu curte muito fazer isso, mas enfim, essa parte é só uma dica.

Novamente, grato! Excelente contéudo!

**Reply by MrPowerGamerBR** [reply]
O problema é ter ideia de conteúdo para postar lmao, mas eu espero ter novas ideias de conteúdo sobre conta PJ/empresa/etc para eu postar no futuro :3 (eu tenho algumas ideias mas ainda nada muito concreto ao ponto de virar um post)
